app/src/main/java/com/cornellappdev/uplift/ui/components/home/HomeCard.kt

```diff
@@ -0,0 +1,171 @@
+    val lastTime =
+        gymexample.hours.get(day)!!.get((gymexample.hours.get(day)!!.size - 1)).end.toString()
+    Box(
+        modifier = Modifier
+            .padding(horizontal = 16.dp, vertical = 12.dp)
+            .fillMaxWidth()
+            .aspectRatio(2F / 1F)
+    )
+    {
+        Card(
+            shape = RoundedCornerShape(12.dp)
+        ) {
+            Column() {
+                Box(modifier = Modifier.weight(3F)) {
+                    AsyncImage(
+                        model = gymexample.imageUrl,
+                        modifier = Modifier.fillMaxWidth(),
+                        contentDescription = "",
+                        contentScale = ContentScale.FillBounds
+                    )
+                    Image(
+                        painterResource(id = if (gymexample.isFavorite()) R.drawable.ic_star_filled else R.drawable.ic_star),
+                        contentDescription = "Star Icon",
+                        modifier = Modifier
+                            // .size(24.dp)
+                            .align(TopEnd)
+                            .padding(top = 12.dp, end = 12.dp)
+                            .align(TopEnd)
+                            .clickable(
+                                interactionSource = MutableInteractionSource(),
+                                indication = null
+                            ) {
+                                gymexample.toggleFavorite()
+                            }
+
+                    )
+                }
+                Column(modifier = Modifier.weight(2F)) {
+                    Row()
+                    {
+                        Spacer(Modifier.width(12.dp))
+                        Text(
+                            text = gymexample.name,
+                            fontSize = 16.sp,
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500)
+
+                        )
+                        Spacer(
+                            modifier = Modifier.weight(1F),
+                        )
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_dumbbell),
+                            contentDescription = "Dumbell",
+                        )
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_bowling_pins),
+                            contentDescription = "Bowling Pins",
+                        )
+                        Spacer(modifier = Modifier.width(12.dp))
+                        Spacer(Modifier.height(4.dp))
+                    }
+                    Row() {
+                        Spacer(Modifier.width(12.dp))
+                        if (determineOpen(gymexample.hours)) Text(
+                            text = "Open",
+                            fontSize = 12.sp,
+                            color = Color.Green,
```

Comment:
Tiny nitpick--the design uses a special green instead of this particular green. In the codebase, you can reference this green with `ACCENT_OPEN` from `Colors.kt`.